jx_sqlite/aggs_table.py

```diff
@@ -301,11 +301,14 @@ def _get(row):
             groupby.append(",\n".join(edge_alias + "." + d for d in domain_names))
             not_groupby.append(",\n".join(edge_alias + "." + d for d in domain_names))
 
-            for k in domain_names:
+            for n, k in enumerate(domain_names):
```

Comment:
this will not work: there should be a double-loop; one to go through all the domain_names, the other to go though all the sort clauses.  But it is good enough for now, I added a test which will catch this error; you can fix it later.